plutus-core/plutus-core/test/Value/Spec.hs

```diff
@@ -86,12 +128,31 @@ prop_insertCoinValidatesToken v =
           BuiltinFailure{} -> property True
           _                -> property False
 
-prop_lookupAfterInsertion :: Value -> ValueAmount -> Property
-prop_lookupAfterInsertion v (ValueAmount amt) =
+prop_insertCoinValidatesQuantityMin :: Value -> Property
+prop_insertCoinValidatesQuantityMin v =
+  forAll gen32BytesOrFewer $ \c ->
+    forAll gen32BytesOrFewer $ \t ->
+      let amt = V.minQuantity - 1
```

Comment:
This isn't really a property-based test, since the random parts are the currencies and tokens, while the quantity is fixed at `minBound - 1`. Try generating more quantities.